sbtStudentCommands/Navigation.scala.template

```diff
@@ -205,6 +213,8 @@ object Navigation {
         withZipFile(state, toProjectName) { () =>
           val src = new sbt.File(new sbt.File(Project.extract(state).structure.root), SSettings.studentifiedBaseFolder)
           val newExerciseSrc = new sbt.File(new sbt.File(Project.extract(state).structure.root), s".cue/$toProjectName")
+          copyExerciseReadme(newExerciseSrc, src)
+          println(s"~~> newExerciseSrc = ${newExerciseSrc} - src = $src")
```

Comment:
Is the `println` intentional?

Reply:
Oops - trailing debug code. Will remove it.